src/app/main/component/user/components/habit/all-habits/all-habits.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface SelectedFilters {
```

Comment:
What the point of creating a new interface if you use it only once?

Reply:
It's used in two places: line 31 and 100 in `src/app/main/component/user/components/habit/all-habits/all-habits.component.ts`.